src/modal/modal.stories.ts

```diff
@@ -164,13 +165,13 @@ class AlertModalStory {
 	@Input() modalTitle: string;
 	@Input() modalContent: string;
 	@Input() buttons: Array<ModalButton>;
-	@Input() size: "xs" | "sm" | "default" | "lg";
```

Comment:
I don't think this should be taken out, because it does show the medium sized modal that's different than the other sizes

Reply:
Huh that doesn't seem to be removed on my end?

Reply:
If there's a specific `"md"` size modal we can add that in

Reply:
I think it just defaults to medium if you provide it with anything other than `xs, sm`, or `lg`. There's no medium class that can be applied

Reply:
Okay, so we _should_ be good then ... We may need to update the story to send a `null` by default though